api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
```

Comment:
It wouldn't work as is - but all current cases can use ._id. Would a note in the method explaining this suffice?  One way to solve this would be to give the Applications Serializer a ListSerializer of its own if we ever extended bulk operations to that endpoint.